components/amorphic/lib/types/IAmorphicAppController.d.ts

```diff
@@ -10,7 +10,7 @@ import { IAmorphicRouteController } from './IAmorphicRouteController';
  * 
  * @misnomer - Functionality that may have confusing nomenclature
  * 
- * @deprecated - Deprecated functionality that is not used anymore
+ * @ deprecated - Deprecated functionality that is not used anymore
```

Comment:
is this intentional?

Reply:
oh you're doing it everywhere, why?

Reply:
Yeah, because this was only supposed to be a glossary basically as a reference to what @deprecated meant. However, VSCode thinks this is referring to the class definition that follows and marks the whole interface as deprecated. (it's not supposed to) I had to make this change everywhere

Reply:
got it